app/Filament/Resources/GameResource/RelationManagers/AchievementSetsRelationManager.php

```diff
@@ -0,0 +1,201 @@
+                                    ->helperText('The name of the subset, such as "Bonus" or "Professor Oak Challenge". Don\'t include the word "Subset".')
+                                    ->required()
+                                    ->rules([
+                                        'not_regex:/\[.*?\]/', // No square brackets.
+                                        'not_regex:/Subset/i', // Can't include the word "Subset".
+                                        'not_regex:/[\x{1F600}-\x{1F64F}]/u',  // No emojis.
+                                    ]),
+
+                                Select::make('type')
+                                    ->options([
+                                        AchievementSetType::WillBeBonus->value => AchievementSetType::Bonus->label(),
+                                        AchievementSetType::WillBeSpecialty->value => AchievementSetType::Specialty->label(),
+                                        AchievementSetType::WillBeExclusive->value => AchievementSetType::Exclusive->label(),
+                                    ])
+                                    ->helperText("
+                                        Bonus loads with any hashes supported by Core.
+                                        Specialty requires a unique hash, but also loads Core.
+                                        Exclusive requires a unique hash, but does not load Core.
+                                        When in doubt, please ask for help.
+                                    ")
+                                    ->required(),
+                            ]),
+                    ])
+                    ->action(function (array $data) use ($game) {
+                        // Add a new game_achievement_set row. The existing row stays
+                        // New row is basically the same as the old row, but the game_id and type are different values.
+                        // It'll use the same achievement_set_id.
+
+                        $legacySubsetGame = Game::find((int) $data['game']);
+                        $legacySubsetAchievementSet = $legacySubsetGame->gameAchievementSets()->core()->first();
+
+                        if ($game->achievementSets()->wherePivot('achievement_set_id', $legacySubsetAchievementSet->id)->exists()) {
+                            Notification::make()
+                                ->danger()
+                                ->title('Error')
+                                ->body('This subset is already associated with the game.')
+                                ->send();
+
+                            return;
+                        }
+
+                        $game->achievementSets()->attach($legacySubsetAchievementSet->id, [
+                            'type' => $data['type'],
+                            'order_column' => 1,
+                            'title' => $data['title'],
+                        ]);
```

Comment:
These rows are being created without created_at or updated_at timestamps.

Reply:
Good catch, fixed in latest.